satellite/satellitedb/migrate.go

```diff
@@ -380,54 +380,52 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 						storage_node_id bytea NOT NULL,
 						PRIMARY KEY ( serial_number_id, storage_node_id )
 					)`,
-					`CREATE TABLE storagenode_bandwidth_rollups (
```

Comment:
I don't know much about migrations, but I think the changes should be in a new version?